ember-file-upload/src/interfaces.ts

```diff
@@ -82,16 +82,6 @@ export enum FileSource {
   Blob = 'blob',
 }
 
-export interface SelectFileModifierSignature {
-  Args: {
-    Named: {
-      filter?: (file: File, files: File[], index: number) => boolean;
-      onFilesSelected?: (files: UploadFile[]) => void;
-    };
-  };
-  Element: HTMLInputElement;
-}
-
```

Comment:
I faced a TypeScript error when trying to use an interface. Inlining fixed it. I have no idea why. Not sure if inlining is acceptable. I also asked on Discord about a better solution: https://discord.com/channels/480462759797063690/484421406659182603/1061264497924382770

Reply:
I think it's fine since it's not imported anywhere else.

When we recently fixed `ember-modifier` deprecations we had to adjust the signature then, also.